src/components/autosuggest-category/autosuggest-category.tsx

```diff
@@ -42,7 +42,7 @@ const AutosuggestCategory: FC<AutosuggestCategoryProps> = ({
                 "autosuggest__text-item--highlight": highlightedIndex === index
               }
             )}
-            key={item}
+            key={item.term + (item.work?.workId || "")}
```

Comment:
Why was this changed?

Reply:
because we were getting a console error that the key needs to be unique :))